src/local-editor-service.ts

```diff
@@ -308,6 +364,16 @@ export class LocalEditorService extends EditorService {
   }
 }
 
+class DatabindingFeature {
```

Comment:
A feature is a high level object that describes something defined in source code.

Added some documentation, PTAL